src/test/java/com/demoqa/pages/VerifyRegistrationPage.java

```diff
@@ -0,0 +1,40 @@
+package com.demoqa.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.interactions.MoveTargetOutOfBoundsException;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.openqa.selenium.support.ui.Wait;
+
+import java.time.Duration;
+import java.util.NoSuchElementException;
+
+import static com.demoqa.util.SingletonBrowser.getInstanceOfSingletonBrowser;
+
+
+public class VerifyRegistrationPage {
```

Comment:
i don't why do you need this class

Reply:
With this page I confirm the registration is completed